.azure-pipelines/client.yml

```diff
@@ -0,0 +1,59 @@
+trigger:
+- master
```

Comment:
Can you add a commented out section here that lists the variables you expect to be inputs from the build definition that uses this definition similar to what @kurtzeborn did here https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/.azure-pipelines/client.yml#L4